web/src/components/molecules/Content/Form/index.tsx

```diff
@@ -59,11 +59,12 @@ const ContentForm: React.FC<Props> = ({
 }) => {
   const t = useT();
   const { Option } = Select;
-
+  const [formValues, setFormValues] = useState<any>();
```

Comment:
@KaWaite , no you can't get the updated form values when changed with `form.getFieldsValue`, so I need the state to keep the value changes updated.

Reply:
It looks like you can get values with `form. getFieldValue`, no?

Reply:
Sorry just read your other comment. But it looks like there is also Form.useWatch (which I know you are using elsewhere).  Since you just need the asset value (passing in to AssetItem, right?) can't you put a watcher on that and then use that variable for the AssetItem's defaultValue?

Reply:
@KaWaite , I considered this approachh before, but also not doable and complicated.
forms are generated dynamically by schema creation.
There may be multiple asset fields each with a dynamic value "UUID" so in the case of `Form.useWatch`, I should iterate over all the ids and check if it's an asset and watch it.